File: Bee-Crowd/Exercises_Python/1066.py
numeros = []
for count in range(0,5):
    numeros.append(int(input()))

pares = []
impar = []
for c in numeros:
    if c % 2 == 0:
        pares.append(c)
    else:
        impar.append(c)

positivo = []
negativo = []
for c in numeros:
    if c > 0:
        positivo.append(c)
    elif c < 0:
        negativo.append(c)

print(len(pares),'valor(es) par(es)')
print(len(impar),'valor(es) impar(es)')
print(len(positivo),'valor(es) positivo(s)')
print(len(negativo),'valor(es) negativo(s)')

File: Bee-Crowd/Exercises_C++/1066.cpp
#include <iostream>
using namespace std;

int main() {

    int n, count,pares,impares,positivos,negativos;
    count = 1;
    pares = 0;
    impares = 0;
    positivos = 0;
    negativos = 0;

    while (count != 6) {
        cin >> n;
        if (n % 2 == 0) {
            pares += 1;
        }
        if (n % 2 != 0) {
            impares += 1;
        }
        if (n > 0) {
            positivos += 1;
        }
        if (n < 0) {
            negativos += 1;
        }
        count += 1;
    }

cout << pares << " valor(es) par(es)" << endl;
cout << impares << " valor(es) impar(es)" << endl;
cout << positivos << " valor(es) positivo(s)" << endl;
cout << negativos << " valor(es) negativo(s)" << endl;
}